acs-integration-tests/testng-predix-cloud-titan.xml

```diff
@@ -19,18 +19,13 @@
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
 <suite name="testAll">
-    <test name="predix-cloud-titan">
+    <test name="predix-cloud-titan" parallel="classes" thread-count="4">
```

Comment:
According to https://wiki.saucelabs.com/display/DOCS/Parallel+Testing+in+Java+with+Maven+and+TestNG:
"...properties that are set in the configuration file will be overridden by their values in the suite definition file. For this reason, it’s best to either define all your configuration properties in the pom.xml file, or in the test suite file, but not in both places."